stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,152 @@ public let NSNotFound: Int = .max
 // NSLocalizedString
 //===----------------------------------------------------------------------===//
 
-/// Returns a localized string, using the main bundle if one is not specified.
+/// Returns the localized version of a string.
+///
+/// - parameter key: An identifying value used to reference a localized string.
+/// - parameter tableName: The name of the table containing the localized string
+///   identified by `key`. This is the prefix of the strings file—a file with
+///   the `.strings` extension—containing the localized values. If `tableName`
+///   is `nil` or the empty string, the `Localizable` table is used. The default
+///   value is `nil`.
+/// - parameter bundle: The bundle containing the table's strings file. The main
+///   bundle is used by default.
+/// - parameter value: A user-visible string to return when the localized string
+///   for `key` can't be found in the table. The default value is the empty
+///   string, which indicates that `key` should be returned.
+/// - parameter comment: A note to the translator describing the context where
+///   the localized string is presented to the user. For example, this can
+///   indicate whether the "Book" is used as a noun or a verb. This distinction
+///   is important when translating to French, where the noun is "Livre" and the
+///   verb is "Réserver".
+///
+/// - returns: A localized version of the string designated by `key` in the
+///   table identified by `tableName`. If the localized string for `key` cannot
+///   be found within the table, the following semantics are used to determine
+///   returned value:
+///     - If `value` is the empty string, `key` is returned.
+///     - If `value` is not the empty string, `value` is returned.
+///
+/// Exporting Localizations
```

Comment:
For this level of header, we use statements that describe the purpose of the following section. Some suggestions for this and the next 2 sections in this comment:

- Use Xcode to Export Localizations
- Choose Meaningful Keys
- Use Only String Literals

Reply:
What are your thoughts on using "Restrict Autosynthesized Localizations" for the last section? I've been going back and forth between that and "Manually Managing Strings Files".